### ISSUE TITLE
[1.12.2] [Thermal Foundation] oreAluminum registration of IE is prevented

### ISSUE THREAD

**Issue by JonasCiss** [CoFH/Feedback#833]
I use Unidict and InstantUnify in my modpack to unify all things. Works for most things. The bauxite ore of IE is not registered in my modpack. So asked there for help. https://github.com/BluSunrize/ImmersiveEngineering/issues/2728 
I was told that it should be registered. So I tested around. 
If Thermal Foundation is not present bauxite ore of IE is registered correctly. If TF is present it prevents the registration of oreAluminum of the bauxite ore. 
I looked into the .cfg but cannot find an option for that. So I think that is a bug.

Forge: 1.12.2 - 14.23.1.2566
TF: ThermalFoundation-1.12.2-2.3.9.4
CoFHCore-1.12.2-4.3.9.2
CoFHWorld-1.12.2-1.1.1.12
IE: 0.12 - 76

**Comment by KingLemming** [comment]
No, TF absolutely does not prevent it. It would be incredibly hostile to prevent another mod from registering their owns ore to the dictionary. This is on IE's end.

**Comment by JonasCiss** [comment]
I believe you but if TF is present there is no registration. And that only happens if TF is present. So how could this be?

**Comment by JonasCiss** [comment]
Maybe there is a conflict? and something is overwriting?

**Comment by JonasCiss** [comment]
There is something I forgot to add. I used UniDict to add additional registration. That works also for everything I added except the bauxite ore block and only if TF is present. No other ores are affected. 

So I have been looking for a fix for weeks. And I found out that there are often conflict with aluminum, aluminium, alumina, bauxite. Are you sure TF has nothing added for that that can cause this issue?
